components/legacy_team_sidebar/components/team_button.tsx

```diff
@@ -118,13 +106,13 @@ class TeamButton extends React.PureComponent {
         const content = (
             <TeamIcon
                 withHover={true}
-                name={this.props.content || displayName}
+                name={this.props.content || displayName || ''}
                 url={teamIconUrl}
             />
         );
 
         let toolTip = this.props.tip || localizeMessage('team.button.name_undefined', 'This team does not have a name');
-        let orderIndicator;
+        let orderIndicator: JSX.Element = React.createElement('div');
```

Comment:
no need for extra div here. null is ok.

Reply:
doing null is throwing the following error
```
Type 'null' is not assignable to type 'Element'
```
and leaving it empty throws the error
```
Variable 'orderIndicator' is used before being assigned
```
any suggestion

Reply:
You can set this to `undefined` instead of null and type it as `JSX.Element | undefined`

Reply:
Fixed it